quantum/matrix_common.c

```diff
@@ -133,11 +133,8 @@ __attribute__((weak)) void matrix_output_unselect_delay(uint8_t line, bool key_p
     matrix_io_delay();
 }
 
-// CUSTOM MATRIX 'LITE'
-__attribute__((weak)) void matrix_init_custom(void) {}
-__attribute__((weak)) bool matrix_scan_custom(matrix_row_t current_matrix[]) {
-    return true;
-}
+__attribute__((weak)) void matrix_init_custom(void);
+__attribute__((weak)) bool matrix_scan_custom(matrix_row_t current_matrix[]);
```

Comment:
```suggestion
void matrix_init_custom(void);
bool matrix_scan_custom(matrix_row_t current_matrix[]);
```

Reply:
Was thinking about this. Makes sense since it's the prototypes. Will test and report back